README.md

```diff
@@ -3,7 +3,9 @@
 GUI building, using the `libui` library: [https://github.com/andlabs/libui](https://github.com/andlabs/libui)
 <br/>
 
-[![Build Status](https://travis-ci.org/joa-quim/Libui.jl.svg?branch=master)](https://travis-ci.org/joa-quim/Libui.jl)
+[![Build Status](https://travis-ci.org/jgoldfar/Libui.jl.svg?branch=master)](https://travis-ci.org/jgoldfar/Libui.jl)
+[![Coverage Status](https://coveralls.io/repos/github/jgoldfar/Libui.jl/badge.svg?branch=master)](https://coveralls.io/github/jgoldfar/Libui.jl?branch=master)
+[![codecov.io](http://codecov.io/github/jgoldfar/Libui.jl/coverage.svg?branch=master)](http://c    odecov.io/github/jgoldfar/Libui.jl?branch=master)
```

Comment:
There's some extra whitespace in the URL, at "c    odecov.io". Probably an accidental press of the tab key?